.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: The version for which the formula will be generated
+        required: true
+      sha256:
+        description: The sha256 of the tuist.zip file associated to the version
+        required: true
+
+permissions:
+  contents: write
+
+env:
+  MISE_EXPERIMENTAL: 1
+
+jobs:
+  release:
+    name: Release
+    runs-on: 'blacksmith-4vcpu-ubuntu-2204'
+    timeout-minutes: 15
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
```

Comment:
What was the reason not to use `actions/checkout@v4` directly?

Reply:
I updated it to `actions/checkout@v4`